source/learn/simple-outlineview.html.markdown

```diff
@@ -0,0 +1,244 @@
+In this tutorial we will create a simple view-based outline view. `CPOutlineView` displays data in hierarchical format. In other programming languages it is referred as Tree-view. Although `CPOutlineView` is a subclass of `CPTableView` and inherits the row and column format, the implementation is slightly different. We will create an organisation chart of a company with the ability to expand and collapse items.
+![](simpleoutlineview/simpleoutlineview.png)
+
+### Create the project
+Run the `capp` command in your project folder:
+
+    :::sh
+    capp gen -l -t NibApplication SimpleOutlineView
+
+This will generate an empty new project using the Nib template.
+
+### Code
+Our data source model will be created in a separate file to keep our code organised  In this way you can easily structure your project around MVC pattern. Create a new file with a name of `EmployeeData.j`.
+
+    :::objj    
+    @import <Foundation/CPObject.j>
+
+    @implementation EmployeeData : CPObject
+    {
+        CPString       _name      @accessors;
+        CPString       _title     @accessors;
+        CPMutableArray _employees @accessors;
+    }
+
+    - (id)initWithDetails:(CPString)aName title:(CPString)aTitle
+    {
+        if (self = [super init])
+        {
+            _name = aName;
+            _title = aTitle;
+            _employees = [CPMutableArray array];
+        }
+        return self;
+    }
+
+    @end
+
+Let’s go back to our AppController.j and import our previously created file:
+`@import "EmployeeData.j"` 
+
+Now, we can create the instance variable using the EmployeeData model and outlets for interaction between AppController and OutlineView.
+
+    :::objj
+    @implementation AppController : CPObject
+    {
+        @outlet CPWindow        theWindow;
+        @outlet CPOutlineView   _outlineView;
+        EmployeeData            ceo; // This will hold our root item
+        EmployeeData            employee; // This will hold the leaf items
+    }
+
+Let's make a windowed application for this tutorial, by setting `setFullPlatformWindow:NO` in the `awakeFromCib` method:
```

Comment:
Well, I'd like to make an application in a window rather than full platform window.